tools/buildsteps/windows/vswhere.bat

```diff
@@ -27,20 +27,14 @@ SET vcvars=no
 SET sdkver=
 
 SET vsver=
-SET toolsdir=%arch%
+REM Current tools are only using x86/win32
+REM ToDo: allow to set NATIVEPLATFORM to allow native tools based on actual native arch (eg x86/x86_64/arm/arm64)
+SET toolsdir=win32
```

Comment:
Want some code to find host arch? NATIVEPLAFORM will be similar.

The cmake side would have to reflect the same logic though, don't know what to do for NATIVEPREFIX

Reply:
Currently, no. Long term sure. I'd rather get something working with the assumptions already in place in the codebase than extend it out to something that really doesn't matter currently

Reply:
OK. Picking one is fine but win32 is making things more complicated though, with 2 different cross-builds. The typical x64 host would target win32 when building the tools and the target arch for Kodi. Are there many win32 build hosts left.

Reply:
https://github.com/xbmc/xbmc/blob/master/project/BuildDependencies/scripts/0_package.native-win32.list

This is the current assumption in our code base.  Whilst these are x86, there is no reason to go out of our way to have multiple different arch types for build tools.

Reply:
I reread earlier in this thread, and i think you have an incorrect idea of the existing system

> Are you sure about removing the option of x64 native tools at this point? Though I don't know your plans for the next steps.

There are no x64 native tools. The current system downloads and extracts the x86 tools into whatever target arch folder (x86/x86_64/uwp_arm/uwp_x64 etc). The tools arent any of those arch types, they are only x86.